### PR TITLE
Issue/add xrv and xrv9k container support

### PR DESCRIPTION
**Pull request by edvgui**
# In this MR

First shot at vrnetlab alternatives for containerlab compatible virtual routers.  
The only kind supported right now are `xrv` and `xrv9k`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
 # containerlab-vms
+
+This project is meant to help building containerlab compatible containers, running virtual routers inside of them.  This has been highly inspired by the vrntelab project.  The working principle is the same as the aforementioned project, but the inner structure of the code has been reviewed.
```
Comment (bartv) [inline README.md]: I think you mean revised here? I would also add something like:

... has been revised to fix stability issues we encountered.

File: makefile-install.include [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#
+# This Makefile can be included by images that need to run an install phase,
```
Comment (bartv) [inline makefile-install.include:5]: I think this file was copied from somewhere (vrnetlab?). We need to attribute that!

File: src/clab_vm_startup/conn_mode/traffic_control.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        super().__init__(ConnectionMode.TC)
+
+    def setup_host(self, host: Host) -> None:
+        """
+        For this connection mode, we need to deploy a script to setup the traffic control rules.
+        """
+        ifup_script = """
+            #!/bin/bash
+
+            TAP_IF=$1
```
Comment (bartv) [inline src/clab_vm_startup/conn_mode/traffic_control.py:41]: Where does this come from?

**Review by bartv — COMMENTED** [review]
First pass with a number of questions

File: src/clab_vm_startup/conn_mode/traffic_control.py (hunk 3 above)
Comment (edvgui) [reply src/clab_vm_startup/conn_mode/traffic_control.py:41]: Here: https://github.com/hellt/vrnetlab/blob/master/common/vrnetlab.py#L328

File: makefile-install.include (hunk 2 above)
Comment (edvgui) [reply makefile-install.include:5]: Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/96d03f6a26eab30f63126f2a2cba0b8be4afb239

File: README.md (hunk 1 above)
Comment (edvgui) [reply README.md]: Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/51586612f99d651afe2c4280d0af16574c66739d

File: makefile-install.include (hunk 2 above)
Comment (bartv) [reply makefile-install.include:5]: Is it different in this repo then the original repo? If so I would link to the original repo

File: src/clab_vm_startup/conn_mode/traffic_control.py (hunk 3 above)
Comment (bartv) [reply src/clab_vm_startup/conn_mode/traffic_control.py:41]: We should attribute this correctly. If it is the same in the orignal repo, link to that.

File: makefile-install.include (hunk 2 above)
Comment (edvgui) [reply makefile-install.include:5]: Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/fcf042eb4e5e65119124ac418af5529ac9c0591e

File: src/clab_vm_startup/conn_mode/traffic_control.py (hunk 3 above)
Comment (edvgui) [reply src/clab_vm_startup/conn_mode/traffic_control.py:41]: Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/6d4269f01d4b064a7bcbe67326233a49a87b1a1f

**Review by bartv — APPROVED** [review]

**Pull request merged**